SYCL/ESIMD/ext_math.cpp

```diff
@@ -467,17 +467,21 @@ int main(void) {
   Pass &= testESIMD<half, 8>(Q);
   Pass &= testESIMD<float, 16>(Q);
   Pass &= testESIMD<float, 32>(Q);
+  Pass &= testSYCL<float, 8>(Q);
+  Pass &= testSYCL<float, 32>(Q);
+  Pass &= testESIMDPow<float, 8>(Q);
+  Pass &= testESIMDPow<half, 32>(Q);
   if (Q.get_backend() != sycl::backend::ext_intel_esimd_emulator) {
     // ESIMD_EMULATOR supports only ESIMD kernels
     Pass &= testSYCL<float, 8>(Q);
     Pass &= testSYCL<float, 32>(Q);
   }
+#ifdef ENABLE_FP64
   Pass &= testESIMDSqrtIEEE<float, 16>(Q);
```

Comment:
Why is this float guarded?

Reply:
Re-enable those IEEE-conformant single precision checks.